src/provisioning-ofono.c

```diff
@@ -1230,10 +1278,9 @@ int provisioning_init_ofono(void)
  * Currently this assumes that only one modem in ofono if need to support more
  * it could be something like below
  */
-#ifdef MULTIMODEM
 	modem_list = g_hash_table_new_full(g_str_hash, g_str_equal,
-						NULL, remove_modem);
-#endif
+						NULL, remove_modems);
+
```

Comment:
`remove_modem` was a better name because it's supposed to free the data associated with ONE modem. If there are N modems in the table, this function is going to be called N times.